### PR TITLE
GET /api/articles?queries

### PR DESCRIPTION
**Pull request by Skahrr**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -280,22 +280,100 @@ describe("POST /api/articles/:article_id/comments", () => {
...
+          expect(article).toEqual(
+            expect.objectContaining({
+              topic: "cats",
+            })
+          );
+        });
+      });
+  });
+  test("status 404: invalid query inputs (sort_by non existent column)", () => {
+    return request(app)
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:364]: this should be a 400 for a bad request if the column does not exist

File: __tests__/app.test.js [hunk 2]
```diff
@@ -280,22 +280,100 @@ describe("POST /api/articles/:article_id/comments", () => {
...
+      .get("/api/articles?sort_by=paco")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Missing data/Wrong datatype");
+      });
+  });
+  test("status 404: invalid query inputs (order input wrong)", () => {
+    return request(app)
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:372]: this should be a 400 for the same reason as above

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
There's a few tests missing here:
200 - valid topic but no article
404 - valid but not existing topic

See comments on existing tests that need amending also. When done please resubmit your PR

**Comment by cavalierswain** [comment]
nice work on the feedback, go ahead and merge when ready

**Comment by Skahrr** [comment]
gracias <3

**Pull request merged**